docs/fundamentals/pipelines.md

```diff
@@ -310,7 +310,11 @@ function) are not transferred into the standalone Task. This means that any pack
 should be imported inside the function itself. 
 
 All imports are automatically logged, and will appear as required packages for the remote Task. This means the 
-`clearml-agent` will install the requested environment automatically
+`clearml-agent` will install the requested environment automatically.
+
+For local imports (e.g. `from my_helper_functions import foo`) you need to make sure these local files are available to 
+the standalone script at runtime. The only way to do this for now is to use the `repo`, `repo_branch` and `repo_commit`
+arguments detailed below. This does mean the helper functions should be committed to that repository at runtime.
```

Comment:
@thepycoder I think the arguments are detailed above rather than below?

Reply:
I moved the section, you're completely right, sorry! :D